### ISSUE TITLE
Handling empty lines in format.py

### ISSUE THREAD

**Issue by khoidt** [cdli-gh/morphology-pre-annotation-tool#28]
The following error occurs when running `mpat -f` with files with empty lines, e.g. [P480078.conll](https://github.com/cdli-gh/morphology-pre-annotation-tool/files/2568317/P480078.conll.txt). 

```
Traceback (most recent call last):
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\lukurkurra\AppData\Local\Programs\Python\Python36\Scripts\mpat.exe\__main__.py", line 9, in <module>
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\cdli_mpa_tool\cli.py", line 53, in main
    check_and_process(pathname, no_output, check, format_conll, verbose)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\cdli_mpa_tool\cli.py", line 19, in check_and_process
    formattor.process()
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\cdli_mpa_tool\format.py", line 43, in process
    self.__file_process()
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\cdli_mpa_tool\format.py", line 36, in __file_process
    line = self.__line_process(line)
  File "c:\users\lukurkurra\appdata\local\programs\python\python36\lib\site-packages\cdli_mpa_tool\format.py", line 15, in __line_process
    if line[0] != '#':
IndexError: string index out of range
```

This can be fixed by replacing `if line[0] != '#':` with `if '#' in line:` (format.py, line 15). However, a somewhat more functional (also corrects encoding, converts CSV to TSV) and (hopefully) stable version of `mpat -f` is now available as `annotation_functions.correct_format()` in [annotation_assistant/scr/main.py](https://github.com/cdli-gh/annotation_assistant/blob/master/scr/main.py).

**Comment by jayanthkmr** [comment]
@khoidt it din't give me this error with that file. I am unable to reproduce the error.

```
jayanth@Jayanths-MacBook-Pro :~/Desktop/gitprojects/morphology-pre-annotation-tool (master)*$ mpat -f -i resources/annotations/P480078.conll 
jayanth@Jayanths-MacBook-Pro :~/Desktop/gitprojects/morphology-pre-annotation-tool (master)*$ 

```

**Comment by jayanthkmr** [comment]
Resolved here - 98923fd. Also, use the latest version of mpat tool - 0.6.0.
